### ISSUE TITLE
JAWS does not identify a math role inside a contenteditable

### ISSUE THREAD

**Issue by mitchellevan** [FreedomScientific/standards-support#799]
### Summary

ARIA `role="math"`:

1. Go to [Math inside a rich text editor - testing user agent support (CodePen)](https://codepen.io/TPG/full/GRrKdop)
2. Use JAWS arrow keys to locate a math element not inside the editor.
3. Under "Enter some text and math", move focus into the editor.
4. Use JAWS arrow keys to locate the math element inside the editor.

HTML `<math>`:

1. Go to TODO
2. Use JAWS arrow keys to locate the math elements. (These are not inside an editor.)
3. Open the attached HTML file in a browser: TODO
4. Move focus into the editor.
5. Use JAWS arrow keys to locate the math elements inside the editor.

### Expected result

Outside of a `contenteditable` JAWS should announce the math content, and optionally the fact that it is a math element.

Inside of a `contenteditable` JAWS should announce the math structure and content, and importantly the fact that it is a math element.
- Announcing the math structure and content allows a user to understand the math together with its surrounding content.
- Informing the user of the fact that it is a math element prompts the user to know what kind of mechanism they should use for editing it.

### Actual result

Outside of a `contenteditable` JAWS announces the math content as expected.

Inside of a `contenteditable` JAWS announces the text of the math content but not its structure, and JAWS does not announce the fact that it is a math element.

### Additional Information

Chrome exposes the math element inside a `contenteditable`. https://bugs.chromium.org/p/chromium/issues/detail?id=1245715

#### JAWS version and build number

JAWS Version 2024.2312.53
 
#### Operating System and version

Windows 10

#### Browser and version:

Chrome 120

**Comment by mitchellevan** [comment]
Chrome and Firefox both report MathML element semantics in devtools accessibility. That said...

If Chrome is partially responsible, please update [Chrome Issue 1245715: JAWS not recognizing role=math element inside contentEditable](https://bugs.chromium.org/p/chromium/issues/detail?id=1245715)

If Firefox is partially responsible, please file a bug for it.
